pallets/dactr/src/lib.rs

```diff
@@ -164,7 +157,6 @@ pub mod pallet {
 				BlockLength::with_normal_ratio(rows, cols, BLOCK_CHUNK_SIZE, NORMAL_DISPATCH_RATIO)
 					.map_err(|_| Error::<T>::BlockDimensionsOutOfBounds)?;
 
-			let _id = Self::next_block_len_proposal_id()?;
```

Comment:
Hey just curious will we need this id for events related to block_length_proposal
CC: @fmiguelgarcia

Reply:
Yes, I think we can remove it.